oms/src/main/java/com/gfg/oms/consignment/response/CreateConsignmentResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.gfg.oms.consignment.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.gfg.oms.Response;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@EqualsAndHashCode(callSuper = true)
```

Comment:
Is this a real need? I think this should have it's own methods

Reply:
Seems that it should be like that (https://projectlombok.org/features/EqualsAndHashCode):

> If applying @EqualsAndHashCode to a class that extends another, this feature gets a bit trickier. Normally, auto-generating an equals and hashCode method for such classes is a bad idea, as the superclass also defines fields, which also need equals/hashCode code but this code will not be generated. By setting callSuper to true, you can include the equals and hashCode methods of your superclass in the generated methods.